core/flyout_horizontal.js

```diff
@@ -275,6 +321,22 @@ Blockly.HorizontalFlyout.prototype.wheel_ = function(e) {
 };
 
 /**
+ * Select a category using the scroll position
+ * @param  {number} pos The scroll position.
+ */
+Blockly.HorizontalFlyout.prototype.selectCategoryByScrollPosition = function(pos) {
+  var scaledPos = pos / this.workspace_.scale;
+  // Traverse the array of scroll positions in reverse, so we can select the lowest
+  // category that the scroll position is below
+  for (var i=this.categoryScrollPositions.length-1; i>=0; i--) {
```

Comment:
Spaces around operators, here and everywhere else.

Reply:
Fixed